todo_app/flask_config.py

```diff
@@ -4,9 +4,6 @@
 class Config:
     def __init__(self):
         """Base configuration variables."""
-        self.SECRET_KEY = os.environ.get('SECRET_KEY')
-        if not self.SECRET_KEY:
-            raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
```

Comment:
This is sensible - we aren't currently using it - but be aware that we will want to bring this back later 🙂